include/zephyr/net/conn_mgr_connectivity.h

```diff
@@ -39,18 +39,26 @@ extern "C" {
 						 NET_MGMT_EVENT_BIT)
 #define _NET_MGMT_CONN_IF_EVENT			(NET_MGMT_IFACE_BIT | _NET_MGMT_CONN_BASE)
 
+/** @endcond */
+
 enum net_event_ethernet_cmd {
```

Comment:
The type name should be updated, looks like a copy-paste error.

Reply:
Good catch! I'm baffled that this never created problems in any of our tests